bedrock/products/templates/products/landing.html

```diff
@@ -15,136 +16,214 @@
   {%- elif LANG == 'en-GB' -%}
     Mozilla’s products — Mozilla (UK)
   {%- else -%}
-    {{ ftl('firefox-products-firefox-beginning', fallback='firefox-products-firefox-is-more-than-a-browser') }} - Mozilla
+    {{ ftl('firefox-products-mozilla-products') }} - Mozilla
   {%- endif -%}
 {%- endblock -%}
 
-{% block page_desc %}{{ ftl('firefox-products-mozillas-family-of-products', fallback='firefox-products-its-a-whole-family-of-products') }}{% endblock %}
+{% block page_desc %}{{ ftl('firefox-products-mozillas-family-of-products') }}{% endblock %}
 
 {% block page_css %}
   {{ css_bundle('protocol-split')}}
   {{ css_bundle('protocol-emphasis-box')}}
   {{ css_bundle('products') }}
 {% endblock %}
 
-{% set android_url = play_store_url('firefox', 'products') %}
-{% set ios_url = app_store_url('firefox', 'products') %}
+{% set fx_android_url = play_store_url('firefox', 'products') %}
+{% set fx_ios_url = app_store_url('firefox', 'products') %}
+{% set fc_android_url = play_store_url('focus', 'products') %}
+{% set fc_ios_url = app_store_url('focus', 'firefox-browsers-mobile-focus') %}
+{% set pocket_android_url = play_store_url('pocket', 'products') %}
+{% set pocket_ios_url = app_store_url('pocket', 'products') %}
+
 {% set referrals = '?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=products' %}
 {% set _entrypoint = 'mozilla.org-products' %}
```

Comment:
If I look correctly, there's no link using the `_entrypoint` — this can be removed entirely.

Reply:
Nope, ignore me — It's used in the included form: https://github.com/mozilla/bedrock/blob/69baed8399e9258ea15020d729c3cda404da1aab/bedrock/mozorg/templates/mozorg/includes/mozilla-account-form-promo.html#L49